Edits for "conventions" and impersonal form (#628)
cf-convention/cf-conventions: ch07.adoc

That would indicate a mean over all longitudes.
Note, however, that if in addition the data variable had a scalar coordinate variable with a **`standard_name`** of **`region`** and a value of **`atlantic_ocean`**, it would indicate a mean over longitudes that lie within the Atlantic Ocean, not all longitudes.

It is recommended that whenever possible, cell bounds should be supplied by giving the variable a dimension of size one and attaching bounds to the associated coordinate variable.

[[climatological-statistics, Section 7.4, "Climatological Statistics"]]
=== Climatological Statistics

Climatological statistics may be derived from corresponding portions of the annual cycle in a set of years, e.g., the average January temperatures in the climatology of 1961-1990, where the values are derived by averaging the 30 Januarys from the separate years.
Portions of the climatological cycle are specified by references to dates within the calendar year.
However, a calendar year is not a well-defined unit of time, because it differs between leap years and other years, and among calendars.

> Nonetheless for practical purposes it may be desirable to compare statistics for months or seasons from different calendars, and to make climatologies from a mixture of leap years and other years.

Reviewer: or "there may be a need to compare"

> Nonetheless for practical purposes it may be desirable to compare statistics for months or seasons from different calendars, and to make climatologies from a mixture of leap years and other years.

Author: ```suggestion
Nonetheless for practical purposes there may be a need to compare statistics for months or seasons from different calendars, and to make climatologies from a mixture of leap years and other years.
```

For example, a commonly-used anomaly is the current temperature at a specific location minus the long-term average temperature there.

CF offers two conventions for describing anomaly data.

> In the remainder of this section and the following two (<<anomalies-norm-data>> and <<anomalies-norm-metadata>>), a general convention for anomalies over any type of coordinate is described, including details about how the norm was calculated.

Author: ```suggestion
The remainder of this section and the following two (<<anomalies-norm-data>> and <<anomalies-norm-metadata>>) describe a general convention for anomalies over any type of coordinate, including details about how the norm was calculated.
```

> In <<temporal-anomalies>>, a legacy convention is described that depends on special standard names.

Reviewer: The remainder of this section and the following two (<<anomalies-norm-data>> and <<anomalies-norm-metadata>>) describe a general convention for anomalies over any type of coordinate, including details about how the norm was calculated.
<<temporal-anomalies>> describes a legacy convention that depends on special standard names.

> In <<temporal-anomalies>>, a legacy convention is described that depends on special standard names.

Author: ```suggestion
<<temporal-anomalies>> describes a legacy convention that depends on special standard names.
```

Using this method means that the norm metadata variable can refer to a subset of elements of the climatological time axis if only some of them are relevant, and it can refer repeatedly to elements of climatological time axis if there is more than one anomaly time referring to a given climatological time.

In the following example, "timestep __i__" means the slice of the data variable along its time dimension with index __i__ (recalling that index numbering starts with 0).
Suppose that __A__ contains monthly anomalies for the months of June, July, and August of 2023 and 2024 with respect to __N__, the 30-year climatological monthly means for 1990-2019.
__N__ has a climatological time axis with a dimension of 12, one for each month January through December.
The anomaly axis is time, and __A__ has a time dimension of 6 (two years times three months).
The first time coordinate of __A__ is June 2023, and the first value of the auxiliary coordinate variable is 5, indicating that timestep 0 of __A__ (June 2023) is the anomaly with respect to the timestep 5 of __N__ (climatological June).

> Since only June, July, and August climatological means are needed in this example, alternatively the climatological time axis of __N__ could be given a dimension of 3, with elements for those three months alone.

Reviewer: I think it would read better as "example, the climatological time axis of __N__ could alternatively be given"

> Since only June, July, and August climatological means are needed in this example, alternatively the climatological time axis of __N__ could be given a dimension of 3, with elements for those three months alone.

Author: ```suggestion
Since only June, July, and August climatological means are needed in this example, the climatological time axis of __N__ could alternatively be given a dimension of 3, with elements for those three months alone.
```

Element **0** of **`month_indices`** is **5**.

This means that element **0** of **`time`**, which is 2023-06-16 (with bounds of 2023-06-01 and 2023-07-01, i.e. the whole of June 2023) is paired with element **5** of **`climatological_time`**, which is the climatology for June 1991-2000.
It indicates that **`delta_tas(0,:,:)`**, where **`:`** means the entire range of the dimension, is the difference between **`tas(0,:,:)`** and **`climatological_tas(5,:,:)`**.

> If instead the climatological time axis is shown with only the three months needed in this example (June, July, and August), the following lines would replace the corresponding ones in the example above:

Reviewer: I suggest, "If instead the climatological time axis comprised only"

> If instead the climatological time axis is shown with only the three months needed in this example (June, July, and August), the following lines would replace the corresponding ones in the example above:

Author: ```suggestion
If instead the climatological time axis comprised only the three months needed in this example (June, July, and August), the following lines would replace the corresponding ones in the example above:
```
